pkg/customresources/certificatestores/certificatestores.go

```diff
@@ -52,24 +62,34 @@ func (c *ActiveCertStores) GetCertsFromStore(ctx context.Context, storeName stri
 	}
 
 	if !hasAccessToStore(ctx, storeName) {
-		return []*x509.Certificate{}, fmt.Errorf("namespace: [%s] does not have access to certificate store: %s", ctxUtils.GetNamespace(ctx), storeName)
+		return []*x509.Certificate{}, errors.ErrorCodeForbidden.WithDetail(fmt.Sprintf("namespace: [%s] does not have access to certificate store: %s", ctxUtils.GetNamespace(ctx), storeName))
```

Comment:
should we add some suggestion on how the customer can get unblocked.

Reply:
sure, added remediation.